src/darker/help.py

```diff
@@ -4,6 +4,8 @@
 
```

Comment:
nit: spacing

Reply:
Sorry, I can't figure out what your diff is – GitHub won't expand it to show more context. Is it about empty lines within the import section?

Here's how `isort` and `black` have sorted and grouped the imports currently:
```python
"""Help and usage instruction texts used for the command line parser"""

from textwrap import dedent

from black import TargetVersion

from darker.formatters import get_formatter_names


def get_extra_instruction(dependency: str) -> str:
```

As I understand it, the empty lines between imports separate them into three sections:
- standard library (`textwrap`),
- 3rd party packages (`black`), and
- "1st party" imports from the package itself (`darker`).

(I tend to instruct linters to consider closely related packages like `darkgraylib` also first party.)